### DISCUSSION TITLE
Ability to run marketplace with multiple NFT contract

### DISCUSSION THREAD

**Discussion by acuarica** [epam/mintgate#86] [General]
Currently, marketplace does not support more than one NFT contract for token ID. Internally, the key is only the token id, but it should be NFT contract account+token id.

- Should we support this use case?

**Comment by acuarica** [comment]
Created issue #92

**Reply by zahhar** [reply]
Jonathan | MintGate — Yesterday at 9:13 PM
_If it's not too complicated or extra effort, that would be nice to support tokens made on other markets. But, it's not a must have as we don't plan to be a major marketplace ourselves_
